tasks/pipeline.py

```diff
@@ -462,7 +462,7 @@ def trigger_child_pipeline(_, git_ref, project_name, variable=None, follow=True)
     if follow:
         print("Waiting for child pipeline to finish...", flush=True)
 
-        wait_for_pipeline(repo, pipeline, pipeline_finish_timeout_sec=2700)
+        wait_for_pipeline(repo, pipeline, pipeline_finish_timeout_sec=7200)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
		# child timeout is 2h = 2*60*60 = 7200sec
        wait_for_pipeline(repo, pipeline, pipeline_finish_timeout_sec=7200)
```

Reply:
Taking into account another comment, let's add various timeouts based on the pipeline we trigger